UI/ui_accessories_text.csv

```diff
@@ -384,9 +384,9 @@ No03980#0,"""Yin & Yang Corsage"""
 No03990#0,"""Sequin Bag A"""
 No03991#0,"""Sequin Bag B"""
 No03992#0,"""Sequin Bag C"""
-No04000#0,"""Ear Type A"""
-No04001#0,"""Ear Type B"""
-No04002#0,"""Ear Type C"""
+No04000#0,"""イヤータイプＡ"""
+No04001#0,"""イヤータイプB"""
+No04002#0,"""イヤータイプＣ"""
```

Comment:
These are default accessories. You need to make your export script ignore these lines.